store/pg_engine.go

```diff
@@ -308,8 +317,19 @@ func migrate(ctx context.Context, d dbdriver.Driver, curVer *semver.Version, mod
 		if err != nil {
 			return fmt.Errorf("failed to read latest data %q, error %w", latestSchemaPath, err)
 		}
-		// We will create the database together with initial schema and data migration.
-		stmt := fmt.Sprintf("CREATE DATABASE %s;\n\\connect \"%s\";\n%s\n%s", databaseName, databaseName, buf, dataBuf)
+
+		var stmt string
+		var createDb bool
+		if strictDb {
+			// User gives only database instead of instance, we cannot create database again.
+			stmt = fmt.Sprintf("\\connect \"%s\";\n%s\n%s", databaseName, buf, dataBuf)
```

Comment:
Do we need to put connect statement for strictDb case, as we've connected to the given database already?